seqr/views/apis/report_api.py

```diff
@@ -342,7 +343,27 @@ def _process_saved_variants(saved_variants_by_family, family_individual_affected
             variant['inheritance_models'] = inheritance_models
             for gene_id in potential_compound_het_gene_ids:
                 potential_com_het_gene_variants[gene_id].append(variant)
+            for tag in variant['discovery_tag_names']:
+                for guid in variant.get('discovery_tag_guids_by_name', {}).get(tag):
+                    potential_mnvs[tag][guid].append(variant)
+        mnv_genes = set()
+        for variants_by_tag_guid in potential_mnvs.values():
+            mnvs = next((variants for variants in variants_by_tag_guid.values() if len(variants) > 2), None)
+            if not mnvs:
+                continue
+            parent_mnv = next((v for v in mnvs if not v.get('populations')), mnvs[0])
+            nested_mnvs = [v for v in mnvs if v.get('populations')]
+            mnv_genes |= {gene_id for variant in nested_mnvs for gene_id in variant['transcripts'].keys()}
+            discovery_notes = 'The following variants are part of the complex structural variant {}: {}'.format(
```

Comment:
this is a bit hard to read, I think it would be better to use an additional helper method for getting the IDs:
```
def _get_nested_variant_name(variant):
    return _get_sv_name(variant) if variant.get('svType') else variant['variantId']
...
discovery_notes = 'The following variants are part of the {variant_type} variant {parent}: {nested}'.format(
    variant_type='complex structural' if parent_mnv.get('svType') else 'multinucleotide',
    parent=_get_nested_variant_name(parent_mnv), 
    nested=', '.join([_get_nested_variant_name(v) for v in nested_mnvs]))
)

```